README.md

```diff
@@ -200,27 +200,33 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    If this is your first time running InstructLab, it's best to start with `ilab config init` to create the environment.
 
    Options:
-   --config PATH  Path to a configuration file.  [default: config.yaml]
+   --config PATH  Path to a configuration file.  [default:
+                  /home/cdoern/.config/instructlab/config.yaml]
```

Comment:
up to you, but maybe better replace particular user names with something more generic.

Reply:
true, will do that

Reply:
I usually add something like <user-name>